README.md

```diff
@@ -0,0 +1,12 @@
+SaC NixOS Packages
+==================
+
+This repository provides NixOS specifications for configuring and installing
+the SaC compiler and standard library. Doing `nix build` should get you going!
```

Comment:
perhaps mention that you need to have flakes enabled? I think it needs flakes

Reply:
Yup!